Origin: commit:26bc6f87c0507d0b457b23548094c475e00e75c9
Author: Tero Mononen <tmo@iki.fi>
Subject: [event-loop] Handle more than 8 events at once

epoll_wait was getting passed a pointer to 64 events,
but was only being told to use up to 8 of those events
at a time because it was using sizeof (pointer) for the
argument specifying the number of events instead of the
actual number of events.

Index: b/src/libply/ply-event-loop.c
===================================================================
--- a/src/libply/ply-event-loop.c
+++ b/src/libply/ply-event-loop.c
@@ -1274,7 +1274,8 @@
        }
 
      number_of_received_events = epoll_wait (loop->epoll_fd, events,
-                                             sizeof (events), timeout);
+                                             PLY_EVENT_LOOP_NUM_EVENT_HANDLERS,
+                                             timeout);
 
      ply_event_loop_handle_timeouts (loop);
 
